web/tests/EducationBenchmarking.Web.E2ETests/Features/SchoolWorkforce.feature

```diff
@@ -0,0 +1,38 @@
+﻿Feature: SchoolWorkforce
+Benchmark the workforce page is showing correct data
+
+    Scenario: download school workforce chart
+        Given I am on workforce page for school with URN '139696'
+        When i click on save as image for school workforce
+        Then school workforce chart image is downloaded
+
+    Scenario: change dimension of school workforce
+        Given I am on workforce page for school with URN '139696'
+        When I change school workforce dimension to 'pupils per staff role'
+        Then the dimension in school workforce dimension dropdown is 'pupils per staff role'
+
+    Scenario: Change dimension in table view for Total number of teachers
```

Comment:
Duplicate "When's" in the scenarios. These should be separate scenarios to test each dimension, or a single scenario just tests the changing of the dimension.

Having they all in a single scenario is difficult to read, and doesn't follow Gherkin guidelines

Reply:
would be resolved in latest commit.